olp-cpp-sdk-core/tests/geo/tiling/TileKeyTest.cpp

```diff
@@ -313,7 +313,7 @@ TEST(TileKeyTest, parentChildRelationIsTransitive) {
 
 // tests for GetNearestAvailableTileKeyLevel(...)
 TEST(TileKeyTest,
-     getNearestAvailableTileKeyLevel_emptyLevelSet_returnedOptionalIsEmpty) {
+     GetNearestAvailableTileKeyLevelEmptyLevelSetReturnedOptionalIsEmpty) {
```

Comment:
Please merge the GetNearestAvailableTileKeyLevel test cases into one by using scopes

```suggestion
     TEST(TileKeyTest, GetNearestAvailableTileKeyLevel) {
     // Use SCOPED_TRACE() for individual scenarios.
     }
```

These test names are too long.